api/seeds/01-roles-and-activities.js

```diff
@@ -1,37 +1,60 @@
-const setupActivities = async activitiesTable => {
-  await activitiesTable.insert({ id: 1, name: 'view-all-apds' });
-  await activitiesTable.insert({ id: 2, name: 'comment-on-apd' });
-  await activitiesTable.insert({ id: 3, name: 'send-apd-to-state' });
-  await activitiesTable.insert({ id: 4, name: 'view-own-apds' });
-  await activitiesTable.insert({ id: 5, name: 'create-apd' });
+const activities = {
+  'view-roles': false,
+  'create-roles': false,
+  'edit-roles': false
 };
 
-const setupRoles = async rolesTable => {
-  await rolesTable.insert({ id: 1, name: 'admin' });
-  await rolesTable.insert({ id: 2, name: 'cms-reviewer' });
-  await rolesTable.insert({ id: 3, name: 'state-submitter' });
+const roles = {
+  admin: false,
+  'cms-reviewer': false,
+  'state-submitter': false
 };
 
-const setupMappings = async mappingTable => {
-  // Admins
-  await mappingTable.insert({ role_id: 1, activity_id: 1 });
-  await mappingTable.insert({ role_id: 1, activity_id: 2 });
-  await mappingTable.insert({ role_id: 1, activity_id: 3 });
-  await mappingTable.insert({ role_id: 1, activity_id: 4 });
-  await mappingTable.insert({ role_id: 1, activity_id: 5 });
-
-  // CMS reviewers
-  await mappingTable.insert({ role_id: 2, activity_id: 1 });
-  await mappingTable.insert({ role_id: 2, activity_id: 2 });
-  await mappingTable.insert({ role_id: 2, activity_id: 3 });
-
-  // State submitters
-  await mappingTable.insert({ role_id: 3, activity_id: 4 });
-  await mappingTable.insert({ role_id: 3, activity_id: 5 });
+const roleToActivityMappings = {
+  admin: [
+    'view-roles',
+    'create-roles',
+    'edit-roles'
+  ],
+  'cms-reviewer': [],
+  'state-submitter': []
+};
+
+// Take the knex object and the table name as arguments, instead of just
+// the knex table object.  The reason is that if you have an insert on
+// the knex table object and then try to do an insert, under the hood it
+// attempts the insert again, which is bad of course, but it also causes
+// a key constraint error.
+const insertAndGetIDs = async (knex, tableName, values) => {
+  const insert = Object.keys(values).map(name => ({ name }));
+  await knex(tableName).insert(insert);
+  const asInserted = await knex(tableName).select('*');
+
+  const idMapping = { };
```

Comment:
for funsies, here's a liner to do this (converting arrays to objects), which i seem to encounter a lot:

`return Object.assign({}, ...asInserted.map(as => ({ [as.name]: as.id })))`